packages/trader/src/App/Components/Elements/PositionsDrawer/__tests__/position-modal-card.spec.tsx

```diff
@@ -53,7 +55,7 @@ const default_mock_store = {
         toggleCancellationWarning: jest.fn(),
     },
     common: {
-        server_time: 123254362145 as unknown as moment.Moment,
+        server_time: moment(new Date()).utc(),
```

Comment:
@kate-deriv will `moment('2023-11-21 10:59:59')` work? in order to avoid these tests to use real date and fail in the midnight? :)

Reply:
Will try, thank you

Reply:
Worked perfectly, thank you